src/components/App.js

```diff
@@ -19,20 +20,24 @@ const App = () => {
   const [selectFilter, setSelectFilter] = useState("all");
   const [isShowCheckbox, setShowCheckbox] = useState(false);
   const [completedAll, setCompletedAll] = useState(false);
+  const [gettingError, setGettingError] = useState("");
+  const [addingError, setAddingError] = useState("");
```

Comment:
Do we really need to use several states for errors? If we will have 10 functions we need 10 states?)